go.mod

```diff
@@ -191,4 +192,5 @@ replace (
 	github.com/CosmWasm/wasmd => github.com/persistenceOne/wasmd v0.0.0-20230615215703-e2229415de74
 	github.com/cosmos/cosmos-sdk => github.com/persistenceOne/cosmos-sdk v0.0.0-20230611231946-790aaf168429
 	github.com/cosmos/ibc-go/v7 => github.com/persistenceOne/ibc-go/v7 v7.0.0-rc0.0.20230612095033-b776ca4647aa
+	github.com/skip-mev/pob => github.com/persistenceOne/pob v1.0.0-lsm
```

Comment:
oh man! i hope cosmos-sdk adopts this if everything is going to be this way

Reply:
@puneet2019 yep... although it's gonna be on a special branch anyway, not sure if this goes into main after successful adoption. Someone jokingly mentioned v0.50.x